src/c++/perf_analyzer/test_command_line_parser.cc

```diff
@@ -1457,6 +1613,12 @@ TEST_CASE("Testing Command Line Parser")
   }
 
   if (check_params) {
+    if (act == nullptr) {
```

Comment:
When would this happen? Is this indicating a bug within the testing code? If so, should that bug be fixed now?

Reply:
The code is just a protection against a case where the user accidentally forgets to specify `check_params = false` in the subcase. This way user gets more helpful message than a random seg fault.

I can refine the error message to mention about `check_params` since the current message seems less helpful.

Reply:
By "user" do you mean the end user of Perf Analyzer? Or do you mean a developer of Perf Analyzer?

I think it's fine to leave in.

Reply:
I meant us, the developers.